### PR TITLE
setup mocha configuration

### PR DESCRIPTION
**Pull request by stefymojica**

### CODE REVIEW

File: test/HelloWorld.test.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const { assert } = require('chai');
+
+describe('Array', () => {
+  describe('#indexOf()', () => {
+    it('should return -1 when the value is not present', function() {
+      assert.equal(-1, [1,2,3].indexOf(4));
+    });
+  });
+});
```
Comment (kliver98) [inline test/HelloWorld.test.js:9]: Add a line break (enter) at the end of the file

File: test/HelloWorld.test.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+
+
+
```
Comment (danielgalvis98) [inline test/HelloWorld.test.js:13]: Don't need all of these. Is just one "Enter" that is needed on line "9"

File: test/HelloWorld.test.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
```
Comment (danielgalvis98) [inline test/HelloWorld.test.js:14]: Remove all the lines from 10 to 14. After that, at the end of line 9 press "Enter". File should have only 10 lines, being the 10th line empty

Comment (stefymojica) [reply test/HelloWorld.test.js]: Fixed, thanks for the comment.

**Review by manuelq12 — APPROVED** [review]
Good job!

**Review by danielgalvis98 — APPROVED** [review]
LGTM. Remember Squash and merge :)

**Pull request merged**